### PR TITLE
Unificated all messages

### PR DESCRIPTION
**Pull request by LanchevychMaxym**

### CODE REVIEW

File: src/components/Notifications/Messages.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    return `Максимальна довжина - ${len} символів`
+}; 
+
+export const minCount = (len:number)=>{
+    return `Мінімальна довжина - ${len} символів`
+}; 
+
+export const maxCount = (len:number)=>{
+    return `Максимальна довжина - ${len} символів`
+};
```
Comment (MarianLabensky) [inline src/components/Notifications/Messages.tsx:27]: You write 2 methods: maxCount and maxLenght with the same returning string

File: src/components/Notifications/Messages.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    return `Мінімальна довжина - ${len} символів`
+}; 
+
+export const maxLength = (len:number)=>{
+    return `Максимальна довжина - ${len} символів`
+}; 
+
+export const minCount = (len:number)=>{
+    return `Мінімальна довжина - ${len} символів`
+};
```
Comment (MarianLabensky) [inline src/components/Notifications/Messages.tsx:23]: You write 2 methods: minCount and minLenght with the same returning string

File: src/models/GllobalValidations/DescriptionValidation.tsx [hunk 3]
```diff
@@ -84,7 +92,7 @@
             String(value).length == 5
                 ? Promise.resolve()
                 : Promise.reject(
-                    `Довжина поштового індексу - 5 символів!`
+                    maxLength(5)
```
Comment (MarianLabensky) [inline src/models/GllobalValidations/DescriptionValidation.tsx:95]: 'Довжина поштового індексу' and 'Максимальна довжина' aren`t the same.

**Review by MarianLabensky — CHANGES_REQUESTED** [review]

File: src/components/Notifications/Messages.tsx (hunk 1 above)
Comment (LanchevychMaxym) [reply src/components/Notifications/Messages.tsx]: Thanks! I've changed it.

File: src/models/GllobalValidations/DescriptionValidation.tsx (hunk 3 above)
Comment (LanchevychMaxym) [reply src/models/GllobalValidations/DescriptionValidation.tsx]: Thanks! I've returned it.

**Review by MykhailoKlapchuk — APPROVED** [review]

**Review by bodnarira — APPROVED** [review]

**Review by MarianLabensky — APPROVED** [review]

**Pull request merged**